app/models/post.rb

```diff
@@ -1,2 +1,22 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_keys: 'author_id'
+  has_many :posts, foreign_keys: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  validates :author_id, presence: true
+  validates :title, presence: true
+  validates :text, presence: true
+
+  after_save :post_counter
```

Comment:
- [ ] @GhostEsso Great work using the after_save [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), however you called the update counter method wrongly instead of ` after_save :post_counter` it should be ` after_save :update_post_counter`